Sources/Optimizely+Decide/OptimizelyDecision.swift

```diff
@@ -0,0 +1,56 @@
+/****************************************************************************
+* Copyright 2020, Optimizely, Inc. and contributors                        *
+*                                                                          *
+* Licensed under the Apache License, Version 2.0 (the "License");          *
+* you may not use this file except in compliance with the License.         *
+* You may obtain a copy of the License at                                  *
+*                                                                          *
+*    http://www.apache.org/licenses/LICENSE-2.0                            *
+*                                                                          *
+* Unless required by applicable law or agreed to in writing, software      *
+* distributed under the License is distributed on an "AS IS" BASIS,        *
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+* See the License for the specific language governing permissions and      *
+* limitations under the License.                                           *
+***************************************************************************/
+    
+import Foundation
+
+/// A decision struct that the SDK makes for a flag key and a user context.
+public struct OptimizelyDecision {
+    
+    /// A variation key of the decision. This value will be nil when decision making fails.
```

Comment:
'A' in this case doesn't look right, same comment for all properties in this struct.
```suggestion
    /// Variation key of the decision. This value will be nil when decision making fails.
```